scripts/deploy.py

```diff
@@ -79,6 +81,20 @@
         print("We've been waiting for an hour. That seems bad. Failing now.")
         sys.exit(1)
 
-    print("Looks good to release! Pushing the tag now.")
+    print("Looks good to release!")
+
+    print("Now uploading to pypi.")
+
+    subprocess.check_output([
+        sys.executable, "setup.py", "sdist"
+    ])
+
+    subprocess.check_output([
+        sys.executable, "-m", "twine", "--config-file=./.pypirc",
+        "upload", "dist/*"
```

Comment:
> Travis specifically does this for us, but I agree that we should do it to just to be safe.

Yeah, I’m going to point to This Entire Pull Request as evidence that we don’t necessarily rely on Travis behaving in a sensible way. 😉